Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1664,14 +1665,17 @@ namespace xr
                             const auto& controllerInfo = sessionImpl.ControllerInfo;
                             auto& gamepadObject = inputSource.GamepadObject;
 
+                            gamepadObject.AxesUsed = 0;
+                            gamepadObject.ButtonsUsed = DEFAULT_CONTROLLER_BUTTONS + SPECIAL_CONTROLLER_BUTTONS;
+
                             // Get interaction data
                             if ((m_impl->TryUpdateControllerBooleanAction(actionResources.HandGetSelectAction, session, gamepadObject.Buttons[controllerInfo.TRIGGER_BUTTON].Pressed)) &&
-                                (m_impl->TryUpdateControllerBooleanAction(actionResources.HandGetSqueezeAction, session, gamepadObject.Buttons[controllerInfo.SQUEEZE_BUTTON].Pressed)))
+                                (m_impl->TryUpdateControllerBooleanAction(actionResources.HandGetSqueezeAction, session, gamepadObject.Buttons[controllerInfo.SPECIAL_BUTTON_1].Pressed)))
                             {
                                 gamepadObject.Buttons[controllerInfo.TRIGGER_BUTTON].Value = (gamepadObject.Buttons[controllerInfo.TRIGGER_BUTTON].Pressed);
                                 gamepadObject.Buttons[controllerInfo.TRIGGER_BUTTON].Touched = (gamepadObject.Buttons[controllerInfo.TRIGGER_BUTTON].Pressed);
-                                gamepadObject.Buttons[controllerInfo.SQUEEZE_BUTTON].Value = (gamepadObject.Buttons[controllerInfo.SQUEEZE_BUTTON].Pressed);
-                                gamepadObject.Buttons[controllerInfo.SQUEEZE_BUTTON].Touched = (gamepadObject.Buttons[controllerInfo.SQUEEZE_BUTTON].Pressed);
+                                gamepadObject.Buttons[controllerInfo.SPECIAL_BUTTON_1].Value = (gamepadObject.Buttons[controllerInfo.SPECIAL_BUTTON_1].Pressed);
```

Comment:
>SPECIAL_BUTTON_1 [](http://example.com/codeflow?start=69&length=16)

nit: Can we rename special to something like unique/custom and avoid having numbers in constexpr names here? 

maybe just call this CUSTOM_HARDWARE_BUTTON for now even if we need to add more in the future